src/user/wikis/Page/Page.js

```diff
@@ -0,0 +1,59 @@
+import React from "react";
+import "./Page.scss";
+import History from "../History/History";
+import ReactMarkdown from "react-markdown";
+import Button from "react-bootstrap/Button";
+import HistoryIcon from "@material-ui/icons/History";
+import EditButton from "@material-ui/icons/EditOutlined";
+import NewPageButton from "@material-ui/icons/DescriptionOutlined";
+
+const Page = (props) => (
+  <div className="page">
+    <div className="page-header">
+      {props.isAdmin && (
+        <div className="wikis-top-controls">
+          <Button
+            variant="light"
+            disabled={props.historyMode}
+            onClick={() => props.handleEditorMode()}
```

Comment:
desctructure props property then use

Reply:
I think too much destructuring makes the code confusing as its not immediately clear where does something come from to whoever is reading the code. I have already used restructuring where it made sense to me, will have al look again and try to incorporate it in more places.

Reply:
Cool then do what you think is correct. Bcoz destructuring never seems confusing to me.
Thanks

Reply:
For your reference code written by some of the best devs of Mozilla https://github.com/mozilla/addons-frontend/blob/master/src/disco/components/Addon/index.js

Reply:
Ok Cool, will destructure everything.

Reply:
Done